pkg/storage/replica.go

```diff
@@ -1640,8 +1640,39 @@ func (r *Replica) assert5725(ba roachpb.BatchRequest) {
 // Range's replicated state. Requests taking this path are ultimately
 // serialized through Raft, but pass through additional machinery whose goal is
 // to allow commands which commute to be proposed in parallel. The naive
-// alternative (submitting requests to Raft one after another, paying massive
-// latency) is only taken for commands whose effects may overlap.
+// alternative, submitting requests to Raft one after another, paying massive
```

Comment:
Note that a lot of this change is merely moving comments from `addWriteCmd` to `tryAddWriteCmd`.